website/content/docs/commands/hcl2_upgrade.mdx

```diff
@@ -79,3 +105,30 @@ working on improving this part of the transformer.
 - `-output-file` - File where to put the hcl2 generated config. Defaults to
   JSON_TEMPLATE.pkr.hcl
 - `-with-annotations` - Adds helper annotations with information about the generated HCL2 blocks.
+
+## User variables using other user variables
```

Comment:
I moved this down the page because it won't be necessary from v1.7.1 but still needed for 1.7.0 and lower